setup.cfg

```diff
@@ -35,7 +35,7 @@ install_requires =
     pyyaml
     sdcflows @ git+https://github.com/nipreps/sdcflows.git@master
     smriprep @ git+https://github.com/poldracklab/smriprep.git@master
-    tedana >= 0.0.5
+    tedana ~= 0.0.9a1
```

Comment:
will this pick up 0.0.9? When pinning RCs in the past, we've had to do this manually, i.e.
`>= 0.0.9a1, < 0.0.10`

Reply:
It was able to grab 0.0.9a1 in the tests, but I wasn't thinking about 0.0.9. Should I switch to `>= 0.0.9a1, < 0.0.10` so it grabs 0.0.9 once it's released, or is it okay to pin to 0.0.9a1?

Reply:
i would lean towards allowing grabbing a more stable version, but i'm not familiar with tedana's release cycle so whatever you think would be best

Reply:
That should be fine. I'm happy to change it.

Reply:
we can always increase/drop the maximum version, thanks!